### PR TITLE
Adding air date 

### PR DESCRIPTION
**Pull request by divakarkd**

### CODE REVIEW

**Comment by divakarkd** [comment]
ok let me check

**Comment by divakarkd** [comment]
I have completed my changes please do review it

File: src/Ombi.Schedule/Jobs/Ombi/NewsletterJob.cs [hunk 1]
```diff
@@ -714,7 +714,8 @@ private async Task ProcessPlexTv(HashSet<PlexEpisode> plexContent, StringBuilder
                         (key, g) => new
                         {
                             SeasonNumber = key,
-                            Episodes = g.ToList()
+                            Episodes = g.ToList(),
+                            EpisodeAirDate =tvInfo.seasons.Where(x=>x.season_number==key).Select(x=>x.air_date)
```
Comment (tidusjar) [inline src/Ombi.Schedule/Jobs/Ombi/NewsletterJob.cs]: `tvInfo` could be null here

File: src/Ombi.Schedule/Jobs/Ombi/NewsletterJob.cs [hunk 2]
```diff
@@ -837,7 +839,8 @@ private async Task ProcessEmbyTv(HashSet<EmbyEpisode> embyContent, StringBuilder
                         (key, g) => new
                         {
                             SeasonNumber = key,
-                            Episodes = g.ToList()
+                            Episodes = g.ToList(),
+                            EpisodeAirDate = tvInfo.seasons.Where(x => x.season_number == key).Select(x => x.air_date)
```
Comment (tidusjar) [inline src/Ombi.Schedule/Jobs/Ombi/NewsletterJob.cs]: `tvInfo` could also be null here

File: src/Ombi.Schedule/Jobs/Ombi/NewsletterJob.cs (hunk 1 above)
Comment (tidusjar) [reply src/Ombi.Schedule/Jobs/Ombi/NewsletterJob.cs]: Can we also fix the spacing on this line?

**Comment by tidusjar** [comment]
It's been merged in

**Pull request closed without merging**